projects/go-lib/src/lib/components/go-side-nav/go-nav-group/go-nav-group.component.ts

```diff
@@ -29,9 +33,20 @@ export class GoNavGroupComponent implements OnInit {
   brandColor: string;
   group: NavGroup;
 
+  // We are using a setter on this ViewChild because of the *ngIf on the element.
+  // This enables us to check whether the element exists before attempting to set its attributes.
+  @ViewChild('navGroupRef') set content(navGroupRef: ElementRef) {
+    if (navGroupRef && this.navItem.attributes) {
+      this.navItem.attributes.forEach((attr: CustomNavAttribute) => {
```

Comment:
Seeing as how this is the same code that exists in the `go-nav-item` as well, what do you think about porting this out into the `GoSideNavService` as a method? That way we can have a test around adding an array of attributes and only have to test that functionality in one spot and we have less overall code?